src/registrar/management/commands/create_federal_portfolio.py

```diff
@@ -51,84 +66,94 @@ def handle(self, agency_name, **options):
+            notes="Auto-generated record",
+            senior_official=so,
         )
 
-        if created:
-            message = f"Created portfolio '{portfolio}'"
-            TerminalHelper.colorful_logger(logger.info, TerminalColors.OKGREEN, message)
+        self.updated_portfolios.add(portfolio)
+        TerminalHelper.colorful_logger(logger.info, TerminalColors.OKGREEN, f"Created portfolio '{portfolio}'")
 
-            if portfolio_args.get("senior_official"):
-                message = f"Added senior official '{portfolio_args['senior_official']}'"
-                TerminalHelper.colorful_logger(logger.info, TerminalColors.OKGREEN, message)
-            else:
-                message = (
-                    f"No senior official added to portfolio '{portfolio}'. "
-                    "None was returned for the reverse relation `FederalAgency.so_federal_agency.first()`"
-                )
-                TerminalHelper.colorful_logger(logger.info, TerminalColors.YELLOW, message)
+        # Log if the senior official was added or not.
+        if portfolio.senior_official:
+            message = f"Added senior official '{portfolio.senior_official}'"
+            TerminalHelper.colorful_logger(logger.info, TerminalColors.OKGREEN, message)
         else:
-            proceed = TerminalHelper.prompt_for_execution(
-                system_exit_on_terminate=False,
-                prompt_message=f"""
-                The given portfolio '{federal_agency.agency}' already exists in our DB.
-                If you cancel, the rest of the script will still execute but this record will not update.
-                """,
-                prompt_title="Do you wish to modify this record?",
+            message = (
+                f"No senior official added to portfolio '{org_name}'. "
+                "None was returned for the reverse relation `FederalAgency.so_federal_agency.first()`"
             )
-            if proceed:
-
-                # Don't override the creator and notes fields
-                if portfolio.creator:
-                    portfolio_args.pop("creator")
-
-                if portfolio.notes:
-                    portfolio_args.pop("notes")
-
-                # Update everything else
-                for key, value in portfolio_args.items():
-                    setattr(portfolio, key, value)
-
-                portfolio.save()
-                message = f"Modified portfolio '{portfolio}'"
-                TerminalHelper.colorful_logger(logger.info, TerminalColors.MAGENTA, message)
-
-                if portfolio_args.get("senior_official"):
-                    message = f"Added/modified senior official '{portfolio_args['senior_official']}'"
-                    TerminalHelper.colorful_logger(logger.info, TerminalColors.MAGENTA, message)
+            TerminalHelper.colorful_logger(logger.info, TerminalColors.YELLOW, message)
 
-        return portfolio
+        return portfolio, True
 
```

Comment:
I had a question about returning True here. Why didn't you use the portfolio itself as a check if the portfolio successfully created or modified?